### PR TITLE
Add explicit Sauce Connect tunnel termination step

### PR DESCRIPTION
**Pull request by bcjordan**

### CODE REVIEW

File: lib/rake/circle.rake [hunk 1]
```diff
@@ -36,5 +36,8 @@ namespace :circle do
         RakeUtils.system_stream_output "bundle exec ./runner.rb --eyes -f #{eyes_features.join(',')} -c ChromeLatestWin7,iPhone -p localhost.code.org:3000 -d localhost.studio.code.org:3000 --circle --parallel 26 --retry_count 3 --html"
       end
     end
+    # Kill Sauce Connect tunnel
+    RakeUtils.system_stream_output 'killall sc'
```
Comment (bcjordan) [inline lib/rake/circle.rake:40]: I may alternatively be able to use [`Process.kill(pid)`](https://ruby-doc.org/core-1.9.3/Process.html#method-c-kill) and [`Process.wait(pid)`](https://ruby-doc.org/core-1.9.3/Process.html#method-c-wait) to do this more cleanly

Comment (bcjordan) [inline lib/rake/circle.rake]: Testing whether this makes my line comment show up.

**Comment by davidsbailey** [comment]
LGTM

**Comment by bcjordan** [comment]
Worked, from end of output:

```
killall sc
14 Sep 20:26:06 - Cleaning up.
sleep 10
14 Sep 20:26:06 - Removing tunnel 83b2d5aad1ee49aeb05d2b11048269fb.
14 Sep 20:26:07 - Number of jobs using tunnel: 1.
14 Sep 20:26:07 - Active jobs using tunnel may fail.
14 Sep 20:26:07 - Waiting for the connection to terminate...
14 Sep 20:26:08 - Connection closed (8).
14 Sep 20:26:08 - Goodbye.
```

**Pull request merged**